Evaluation/Input/Content/Section2.2_Data.md

```diff
@@ -1,40 +1,19 @@
-### In vitro / physico-chemical Data <a id="invitro-and-physico-chemical-data"></a>
+### 2.2.1	In vitro / physico-chemical Data
 
-A literature search was performed to collect available information on physicochemical properties of dapagliflozin. The obtained information from literature is summarized in the table below. 
+A literature search was performed to collect available information on physicochemical properties of 7E3. The obtained information from literature is summarized in the table below. 
 
-| **Parameter**   | **Unit** | **Value** | Source                                     | **Description**                                 |
-| :-------------- | -------- | --------- | ------------------------------------------ | ----------------------------------------------- |
-| MW              | g/mol    |           | [Dummy YEAR](#main-references)               | Molecular weight                                |
-| pK<sub>a</sub>  |          |           | [DrugBank DB06292](#main-references)         | Acid dissociation constant                      |
-| Solubility (pH) |          |           | [Dummy YEAR](#main-references)               | Aqueous Solubility, FaSSIF, ...                 |
-| logP            |          |           | [Dummy YEAR](#main-references) (experimental) | Partition coefficient between octanol and water |
-| fu              | %        |           | [Dummy YEAR](#main-references)                | Fraction unbound in plasma                      |
-| B/P ratio       |          |           | [Dummy YEAR](#main-references)                | Blood to plasma ratio                           |
-| ...             |          |           |                                            | ....                                            |
+| **Parameter** | **Unit** | **Value** | Source                       | **Description**                                              |
+| :------------ | -------- | --------- | ---------------------------- | ------------------------------------------------------------ |
+| MW            | g/mol    | 150000    | [Lobo 2004](#5-references)   | Molecular weight                                             |
+| r             | nm       | 5.34      | [Taylor 1984](#5-references) | Hydrodynamic solute radius                                   |
+| Kd (FcRn)     | µM       | 0.75      | [Zhou 2003](#5-references)   | Dissociation constant for binding of a murine IgG1 antibody to murine FcRn at pH 6 |
 
-### Clinical Data  <a id="clinical-data"></a>
+### 2.2.2	PK Data
```

Comment:
never remove predefined anchors from the subchapters.
Due to removing of `<a id="clinical-data"></a>` here all cross-references to this subchapter in the report became invalid
```
Evaluation/report/report.html
  error: bad link report.html#clinical-data

Error: bad anchors:
  report.html#clinical-data

Found 0 bad links
Found 2 bad anchors
```

Reply:
Also here: too many spaces in the title after **2.2.2**

Reply:
fixed now